src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorResolver.cs

```diff
@@ -89,7 +97,7 @@ public IEnumerable<TagHelperDescriptor> Resolve([NotNull] TagHelperDescriptorRes
                         directiveDescriptor.Location,
                         Resources.FormatTagHelperDescriptorResolver_EncounteredUnexpectedError(
                             "@" + directiveName,
-                            directiveDescriptor.LookupText,
+                            directiveDescriptor.DirectiveText,
```

Comment:
`directiveDescriptor` is now used for `@addTagHelper`, `@removeTagHelper`, `@tagHelperPrefix` so `LookupText` didn't make sense.